nyaa/views/account.py

```diff
@@ -223,6 +223,48 @@ def profile():
     return flask.render_template('profile.html', form=form)
 
 
+@bp.route('/trusted/request', methods=['GET', 'POST'])
+def request_trusted():
+    if not flask.g.user:
+        return flask.redirect(flask.url_for('account.login'))
+    trusted_form = None
+    deny_reasons = []
+    if flask.request.method == 'POST':
+        trusted_form = forms.TrustedForm(flask.request.form)
+        if trusted_form.validate():
+            ta = models.TrustedApplication()
+            ta.submitter_id = flask.g.user.id
+            ta.why_want = trusted_form.why_want_trusted.data
+            ta.why_give = trusted_form.why_give_trusted.data
+            db.session.add(ta)
+            db.session.commit()
+            flask.flash('Your trusted application has been submitted.', 'success')
+            return flask.redirect('/trusted')
+    else:
+        if flask.g.user.is_trusted:
+            deny_reasons.append('You already are trusted.')
+        if not flask.g.user.satisfies_trusted_reqs:
+            deny_reasons.append('You do not satisfy the minimum requirements.')
+        if (models.TrustedApplication.query.
```

Comment:
`.first()` is fine instead of `.count() > 0`, although if you chose this for performance, :woman_shrugging:.